### PR TITLE
Users/abmodi/master

### PR DESCRIPTION
**Pull request by abhinitmodi**
Bug fixes and more

### CODE REVIEW

File: Tasks/DeployAzureResourceGroup/AzureResourceManagerHelper.ps1 [hunk 1]
```diff
@@ -134,11 +134,11 @@ function Get-Resources
                 }
         
                 #Adding WinRMHttp port property
-                if([string]::IsNullOrEmpty($winRmHttpPortMap[$resource.Name]) -eq $false)
-                {
-                    $property = New-Object Microsoft.VisualStudio.Services.DevTestLabs.Model.PropertyBagData($false, $winRmHttpPortMap[$resource.Name])
-                    $propertyBag.Add("WinRM_Http", $property)
-                }
+                #if([string]::IsNullOrEmpty($winRmHttpPortMap[$resource.Name]) -eq $false)
```
Comment (pankajkhanzode) [inline Tasks/DeployAzureResourceGroup/AzureResourceManagerHelper.ps1]: Can we make this change as - 
if we find both HTTPS and HTTP - add HTTPS only, else add whatever is present.

File: Tasks/DeployAzureResourceGroup/AzureResourceManagerHelper.ps1 [hunk 2]
```diff
@@ -134,11 +134,11 @@ function Get-Resources
...
+                #{
+                #    $property = New-Object Microsoft.VisualStudio.Services.DevTestLabs.Model.PropertyBagData($false, $winRmHttpPortMap[$resource.Name])
+                #    $propertyBag.Add("WinRM_Http", $property)
+                #}
 
                 #Adding WinRMHttps port property
                 if([string]::IsNullOrEmpty($winRmHttpsPortMap[$resource.Name]) -eq $false)
```
Comment (pankajkhanzode) [inline Tasks/DeployAzureResourceGroup/AzureResourceManagerHelper.ps1]: And when we add port -can u also add Microsoft-Vslabs-MG-WinRMProtocol which would be either HTTP or HTTPS

**Comment by pankajkhanzode** [comment]
Signed off.

**Comment by NikhilRanjanKumar** [comment]
Signed Off

**Comment by AnubhavAggarwal** [comment]
merging on behalf of abmodi

**Pull request merged**